.buildkite/e2e/nightly-main-matrix.yaml

```
@@ -4,17 +4,16 @@
     E2E_PROVIDER: gke
   mixed:
     - E2E_STACK_VERSION: "6.8.23"
-    - E2E_STACK_VERSION: "7.13.4"
-    - E2E_STACK_VERSION: "7.14.2"
-    - E2E_STACK_VERSION: "7.15.2"
     - E2E_STACK_VERSION: "7.16.3"
     - E2E_STACK_VERSION: "7.17.8"
     - E2E_STACK_VERSION: "8.0.1"
     - E2E_STACK_VERSION: "8.1.3"
     - E2E_STACK_VERSION: "8.2.3"
     - E2E_STACK_VERSION: "8.3.3"
     - E2E_STACK_VERSION: "8.4.3"
-    - E2E_STACK_VERSION: "8.7.0-SNAPSHOT"
+    - E2E_STACK_VERSION: "8.5.3"
+    - E2E_STACK_VERSION: "8.6.2"
```

Maybe I'm missing something, should we not also add `8.7.0`?
```suggestion
    - E2E_STACK_VERSION: "8.6.2"
    - E2E_STACK_VERSION: "8.7.0"
```

---

Historically we don't test the current version of the stack (`Makefile:E2E_STACK_VERSION ?= 8.7.0`) in the testsuite runs that test all versions of the stack because it is already test in all other testsuite runs.